cmd/cloud/backup.go

```diff
@@ -100,7 +99,7 @@ var backupListCmd = &cobra.Command{
 					backup.InstallationID,
 					string(backup.State),
 					backup.ClusterInstallationID,
-					utils.TimeFromMillis(backup.RequestAt).Format("2006-01-02 15:04:05 -0700 MST"),
+					model.TimeFromMillis(backup.RequestAt).Format("2006-01-02 15:04:05 -0700 MST"),
```

Comment:
A super minor PR nitpick: moving this helper function to `model` is a change I support, but it would've been nice to see it in its own commit so that I could view the meat of the change surrounding the new command separately from this rename